Testing/functional/tests/cypress/integration/ui/errors/tooManyRequestsAlerts.js

```diff
@@ -0,0 +1,461 @@
+    "[data-testid=dateOfVaccineInput] [data-testid=formSelectDay]";
+const collectionDateYearSelector =
+    "[data-testid=dateOfCollectionInput] [data-testid=formSelectYear]";
+const collectionDateMonthSelector =
+    "[data-testid=dateOfCollectionInput] [data-testid=formSelectMonth]";
+const collectionDateDaySelector =
+    "[data-testid=dateOfCollectionInput] [data-testid=formSelectDay]";
+
+const fullyVaccinatedPhn = "9735361219";
+const fullyVaccinatedDobYear = "1994";
+const fullyVaccinatedDobMonth = "June";
+const fullyVaccinatedDobDay = "9";
+const fullyVaccinatedDovYear = "2021";
+const fullyVaccinatedDovMonth = "January";
+const fullyVaccinatedDovDay = "20";
+
+function enterVaccineCardPHN(phn) {
+    cy.get("[data-testid=phnInput]")
+        .should("be.visible", "be.enabled")
+        .clear()
+        .type(phn);
+}
+
+function clickVaccineCardEnterButton() {
+    cy.get("[data-testid=btnEnter]").should("be.enabled", "be.visible").click();
+}
+
+describe("Authenticated Vaccine Card Downloads", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        cy.intercept("GET", "**/AuthenticatedVaccineStatus?hdid=*", {
+            statusCode: 429,
+        });
+        cy.enableModules(["Immunization", "VaccinationStatus"]);
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak,
+            "/covid19"
+        );
+
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+
+    it("Save As PDF Unsuccessful Response: Too Many Requests", () => {
+        cy.intercept("GET", "**/AuthenticatedVaccineStatus/pdf?hdid=*", {
+            statusCode: 429,
+        });
+        cy.enableModules([
+            "Immunization",
+            "VaccinationStatus",
+            "VaccinationStatusPdf",
+            "VaccinationExportPdf",
+        ]);
+        cy.login(
+            Cypress.env("keycloak.username"),
+            Cypress.env("keycloak.password"),
+            AuthMethod.KeyCloak,
+            "/covid19"
+        );
+
+        cy.get("[data-testid=save-dropdown-btn] .dropdown-toggle")
+            .should("be.enabled", "be.visible")
+            .click();
+        cy.get("[data-testid=save-as-pdf-dropdown-item]")
+            .should("be.visible")
+            .click();
+        cy.get("[data-testid=genericMessageSubmitBtn]").click();
+
+        cy.get("[data-testid=too-many-requests-warning]").should("be.visible");
+    });
+});
+
+describe("Public COVID-19 Test Results", () => {
+    it("Unsuccessful Response: Too Many Requests", () => {
+        const phn = "9735361219 ";
+        const dobYear = "1994";
+        const dobMonth = "June";
+        const dobDay = "9";
+        const collectionDateYear = "2021";
+        const collectionDateMonth = "January";
+        const collectionDateDay = "20";
+
+        cy.enableModules(["Laboratory", "PublicLaboratoryResult"]);
+        cy.logout();
```

Comment:
I saw this being done for the public pages in the tests so I kept it here since it is likely making sure there is no leftover login.  I can remove it if that is wrong.